tests/resources/test_render_run_dq_main_sql_expected.sql

```diff
@@ -125,6 +125,33 @@ SELECT
   ON
     zero_record.rule_binding_id = data.rule_binding_id
 
+    UNION ALL
+    SELECT
+    CURRENT_TIMESTAMP() AS execution_ts,
+    'T2_DQ_1_EMAIL' AS rule_binding_id,
+    'CUSTOM_SQL_LENGTH_LE_PARAMETRIZED' AS rule_id,
+    
+    '<your_gcp_project_id>.<your_bigquery_dataset_id>.contact_details' AS table_id,
+    'value' AS column_id,
+    data.value AS column_value,
+    CAST(NULL AS STRING) AS dimension,
+    CASE
+
+      WHEN value IS NULL THEN CAST(NULL AS BOOLEAN)
+      WHEN LENGTH( value ) <= 0 THEN TRUE
```

Comment:
same comment here, should it not be `WHEN LENGTH( value ) <= 30`?

Reply:
I wanted to have a different test for 0 valued arguments. This resource is part of is a unit test. The `dq_rules_configs.yml` file is part of a integration test so there I wanted it to match the output of a non-parametrized rule.